packages/drivers/local-driver/package.json

```diff
@@ -134,7 +134,6 @@
 		"@fluidframework/telemetry-utils": "workspace:~",
 		"events": "^3.1.0",
 		"jsrsasign": "^11.0.0",
-		"url": "^0.11.0",
```

Comment:
I love it. This was something I was dealing with by a rename to avoid any uses of node features in runtime product. Full removal is nice to see.

Reply:
IntelliSense was a little odd, but it does show properly coming from either DOM or node. So, we appear all good. Back to fully loving it.